install.ps1

```diff
@@ -257,9 +266,9 @@ if (-not $noChecks.IsPresent) {
     }
 
     # Internet connectivity checks
-    Test-WebConnection "google.com"
-    Test-WebConnection "github.com"
-    Test-WebConnection "raw.githubusercontent.com"
+    Test-WebConnection 'microsoft.com'
```

Comment:
This change is now removing `raw.githubusercontent.com`

Reply:
Sorry for the minor typo, this is fixed.